chapters/3-measurement.tex

```diff
@@ -931,3 +931,22 @@ \section{Doing power calculations for research design}
 or if results seem to depend dramatically on the outcomes for a small number of individuals,
 randomization inference will flag those issues before the experiment is fielded
 and allow adjustments to the design to be made.
+
+\section{Looking ahead}
+This chapter provided a framework for you
+develop a data map that documents the design of your study.
```

Comment:
```suggestion
develop a data map that documents your data needs and 
how they relate to the design of your study.
```
Something like this is needed. it is not the design that it documents...

Reply:
Hmm, I don't agree, I think the point of master datasets and research variables _is_ to document the study design! or at least its execution.

Reply:
Softly in favor of Kris's suggestion, i do think of these as providing a framework for linking all the data to the research questions

Reply:
Ok -- edited and adopted